web/src/main/java/org/cbioportal/web/util/ClinicalDataEnrichmentUtil.java

```diff
@@ -81,16 +83,17 @@
                 }
             }
 
-            Double totalCount = transposeDataCollection
+            Supplier<Stream<Double>> valuesStreamSupplier = () -> transposeDataCollection
                     .values()
                     .stream()
-                    .flatMap(collection -> collection.stream())
-                    .mapToDouble(x -> (Double) x)
-                    .sum();
+                    .flatMap(Collection::stream)
+                    .map(x -> (Double) x);
+
+            List<Double> distinctValues = valuesStreamSupplier.get().distinct().collect(Collectors.toList());
 
             // perform test only if there are more than one group and
             // total count across all groups in greater than 0
```

Comment:
this line seems outdated, could we update this?

Reply:
thanks! updated